src/System.Threading.Tasks/pkg/aot/System.Threading.Tasks.pkgproj

```diff
@@ -7,5 +7,11 @@
     <PreventImplementationReference>true</PreventImplementationReference>
   </PropertyGroup>
 
+  <ItemGroup>
```

Comment:
Didn't you have to add a placeholder on the runtimes/any package as you did in Globalization?